docs/development.md

```diff
@@ -47,7 +55,7 @@ code as it can help with enabling a faster iteration.
 
 The following command builds and runs an Oak Development Server:
 
-`./scripts/run_server_dev`
+`./scripts/run_server_dev --config="$PWD/bazel-client-bin/examples/hello_world/config/config.bin"`
```

Comment:
It looks like this `--config` flag may be confused with the `--config=tsan` flag below, but also in general it seems quite generic. Should we consider renaming to `--application` or similar?

Reply:
Done